### ISSUE TITLE
Linux package to support different distro

### ISSUE THREAD

**Issue by tuapuikia** [lensapp/lens#475]
@jakolehm Is it possible to provide an official deb or rpm package for Lens? Some users might prefer to have the application installed using the distribution package manager.

** What would you like to be added**

DEB and RPM repository to support apt or yum installation.

**Comment by debMan** [comment]
I have many problems with `snapcraft`. It makes messy my OS and usually have problems.
Sometimes the Lens not start, neither on my terminal nor on `gnome-desktop`. No log output when run `kontena-lens` on the CLI. 
In my former distro, (= Ubuntu 18.04 LTS) the `apparmor` prevents starting Lens, so I've fixed it by defining a function in my `bash`
``` bash
snap-fix ()
{
    sudo apparmor_parser -r /etc/apparmor.d/*snap-confine*
    sudo apparmor_parser -r /var/lib/snapd/apparmor/profiles/snap-confine*
}
```

Please take it out from **`snap` HELL**.

**Comment by tuapuikia** [comment]
Snap is working fine on my Ubuntu 18.04 and 20.04 distro.

**Comment by jakolehm** [comment]
Fixed in #2053 .
